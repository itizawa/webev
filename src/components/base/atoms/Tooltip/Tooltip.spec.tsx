import renderer from 'react-test-renderer';
import { Tooltip } from './index';

test('Tooltip', () => {
  const tree = renderer
    .create(
      <Tooltip text="Tooltip" placement="bottom">
        target
      </Tooltip>,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});

test('Tooltip without placement', () => {
  const tree = renderer.create(<Tooltip text="Tooltip">target</Tooltip>).toJSON();

  expect(tree).toMatchSnapshot();
});

test('Tooltip with fade', () => {
  const tree = renderer
    .create(
      <Tooltip text="Tooltip" fade>
        target
      </Tooltip>,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
