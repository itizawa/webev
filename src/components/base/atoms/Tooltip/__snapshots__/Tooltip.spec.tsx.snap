// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip 1`] = `
<div>
  target
</div>
`;

exports[`Tooltip with fade 1`] = `
<div>
  target
</div>
`;

exports[`Tooltip without placement 1`] = `
<div>
  target
</div>
`;
